packages/cli/doc-plugin-preview/src/index.ts

```diff
@@ -158,7 +162,7 @@ export const pageType = "blank";
       `../static/${isMobile ? 'mobile' : 'web'}.css`,
     ),
     addSSGRoutes() {
-      return isMobile ? demoRoutes : [];
+      return demoRoutes;
```

Comment:
It's no need to add ssg routes in web mode

Reply:
In web mode, it may have mobile preview code, if not, it will be empty array

Reply:
So in `codeToDemo.ts`, the demoRoutes should only include mobile demo routes, instead of all demo routes.

Reply:
Yes, I have fixed it.